### PR TITLE
Added deuteron PDG to recognise by the task

### PR DESCRIPTION
**Pull request by neelimaagrawal**

### CODE REVIEW

File: PWGCF/FemtoDream/femtoDreamPairTaskTrackTrack.cxx [hunk 1]
```diff
@@ -116,6 +117,9 @@ struct femtoDreamPairTaskTrackTrack {
 
   void init(InitContext&)
   {
+    TDatabasePDG::Instance()->AddParticle("deuteron", "deuteron", 1.8756134, kTRUE, 0.0, 3, "Nucleus", 1000010020);
+    TDatabasePDG::Instance()->AddAntiParticle("anti-deuteron", -1000010020);
```
[the remark this answers is not in the document]
Comment (victor-gonzalez) [reply PWGCF/FemtoDream/femtoDreamPairTaskTrackTrack.cxx:121]: I will approve it for the time being and if centralized processing is decided the change will be rolled back

**Review by victor-gonzalez — APPROVED** [review]

**Comment by aalkin** [comment]
PDG database should be used through `Service<O2DatabasePDG> pdg;`, that has all the central customizations.

**Comment by victor-gonzalez** [comment]
Thanks Anton!!
So this PR is not then needed!!!

**Comment by aalkin** [comment]
Well, the task needs to be modified still, adding the O2DatabasePDG, but adding the particle should not be required.

**Comment by victor-gonzalez** [comment]
You are right!
I meant this PR with this title and this content

**Comment by victor-gonzalez** [comment]
@neelimaagrawal I will close it now to prevent someone else to merge it
Could you please take care of open a new PR in the line of what Anton suggested?
Thanks!

**Pull request closed without merging**